### PR TITLE
Add documentation for counter component

### PR DESCRIPTION
**Pull request by fabaff**
**Description:**
Add documentation for counter component.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#9146

### CODE REVIEW

File: source/_components/counter.markdown [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+### {% linkable_title Media control services %}
+Available services: `increment`, `decrement`, and `reset`.
+
+#### {% linkable_title Service `counter/increment` %}
+
+Increments the counter with 1 or the given value for the steps.
+
+| Service data attribute | Optional | Description |
+| ---------------------- | -------- | ----------- |
+| `entity_id`            |      no  | Name of entity to take the action, e.g. `counter.count0`. |
```
Comment (frenck) [inline source/_components/counter.markdown]: **Spelling & Grammar**:
- `Name of entity` -> `Name of the entity``
- `e.g.` -> `e.g.,` (missing comma)
- `take the action` -> `take action`

File: source/_components/counter.markdown [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+#### {% linkable_title Service `counter/decrement` %}
+
+Decrements the counter with 1 or the given value for the steps.
+
+| Service data attribute | Optional | Description |
+| ---------------------- | -------- | ----------- |
+| `entity_id`            |      no  | Name of entity to take the action, e.g. `counter.count0`. |
```
Comment (frenck) [inline source/_components/counter.markdown]: Same spelling & grammar issues as mentioned above.

File: source/_components/counter.markdown [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+#### {% linkable_title Service `counter/reset` %}
+
+With this service the counter is reset to its initial value.
+
+| Service data attribute | Optional | Description |
+| ---------------------- | -------- | ----------- |
+| `entity_id`            |      no  | Name of entity to take the action, e.g. `counter.count0`. |
```
Comment (frenck) [inline source/_components/counter.markdown]: Same spelling & grammar issues as mentioned in my first comment.

File: source/_components/counter.markdown [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+
+Decrements the counter with 1 or the given value for the steps.
+
+| Service data attribute | Optional | Description |
+| ---------------------- | -------- | ----------- |
+| `entity_id`            |      no  | Name of entity to take the action, e.g. `counter.count0`. |
+
+#### {% linkable_title Service `counter/reset` %}
+
+With this service the counter is reset to its initial value.
```
Comment (frenck) [inline source/_components/counter.markdown]: **Spelling & Grammar**:
- Missing comma after `service` -> `With this service, the counter`

File: source/_components/counter.markdown [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+
+| Service data attribute | Optional | Description |
+| ---------------------- | -------- | ----------- |
+| `entity_id`            |      no  | Name of entity to take the action, e.g. `counter.count0`. |
+
+
+### {% linkable_title Use the service %}
+
+Select <img src='/images/screenshots/developer-tool-services-icon.png' alt='service developer tool icon' class="no-shadow" height="38" /> **Services** from the **Developer Tools**. Choose **counter** from the list of **Domains**, select the **Service**, enter something like the sample below into the **Service Data** field, and hit **CALL SERVICE**.
```
Comment (frenck) [inline source/_components/counter.markdown]: Is the image missing?

Comment (frenck) [reply source/_components/counter.markdown]: Nevermind, my bad, you can ignore this one 😅

File: source/_components/counter.markdown (hunk 1 above)
Comment (fabaff) [reply source/_components/counter.markdown]: Thanks

File: source/_components/counter.markdown (hunk 5 above)
Comment (fabaff) [reply source/_components/counter.markdown]: This is an issue with Github preview. Will only be visible after deployment.

Comment (frenck) [reply source/_components/counter.markdown]: I'm actually running branches locally when reviewing... 
Nevertheless, it was an issue on my end 👍

**Comment by frenck** [comment]
Thx @fabaff 🥇

**Pull request merged**